Providers/IntercomProvider.m

```diff
@@ -44,6 +44,10 @@ - (void)identifyUserWithID:(NSString *)userID andEmailAddress:(NSString *)email
     }
 }
 
+- (void)setUserProperty:(NSString *)property toValue:(NSString *)value {
+    [Intercom updateUserWithAttributes:@{[property lowercaseString]: value}];
```

Comment:
they ignore it. There is a way to add custom properties (that is different from this one), but I am not sure it's required (nor I don't want to hold the entire list of options in the code)